Sources/ApolloCodegenLib/ApolloExtension.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public struct ApolloExtension<Base> {
+  public let base: Base
+}
+
+protocol ApolloCompatible {
```

Comment:
I suspect this needs to be `public protocol ApolloCompatible`

Reply:
I fixed it. Some of CI tests are still failing but Im not sure why.

Reply:
oh wait Swift Code Generation failure is my fault.